pandas/core/generic.py

```diff
@@ -5783,7 +5784,7 @@ def pct_change(self, periods=1, fill_method='pad', limit=None, freq=None,
                                   **kwargs)) - 1)
         if freq is None:
             mask = isnull(_values_from_object(self))
-            np.putmask(rs.values, mask, np.nan)
```

Comment:
I reverted back to `rs.mask(mask, np.nan, inplace=True)` as `rs[mask] = np.nan` fails the test test_pct_change